.github/templates/model_regression_test_results.tmpl

```diff
@@ -13,6 +13,8 @@ The accuracy_tpl template returns data for the accuracy field.
 {{ define "accuracy_tpl" -}}
 {{- if has .master "accuracy" -}}
 {{ printf "%.4f" .branch.accuracy }} ({{ printf "%.2f" (.master.accuracy | math.Sub .branch.accuracy) }})
+{{- else if (index .master "micro avg") "f1-score" -}}
```

Comment:
Just looking at the code above and below, shouldn't we combine the `accuracy_tpl` and `micro_avg_tpl` into one since you now anyways look under micro avg if accuracy is not present?

Reply:
Good point! I did a small refactor for the model_regression_test_results temple.